AlphaWallet/InCoordinator.swift

```diff
@@ -891,13 +870,8 @@ extension InCoordinator: PaymentCoordinatorDelegate {
     }
 
     func didCancel(in coordinator: PaymentCoordinator) {
-        switch coordinator.flow {
-        case .request:
-            coordinator.navigationController.setNavigationBarHidden(true, animated: false)
-            coordinator.navigationController.popToRootViewController(animated: true)
-        case .send:
-            coordinator.navigationController.dismiss(animated: true)
-        }
+        coordinator.navigationController.setNavigationBarHidden(true, animated: false)
```

Comment:
After sending a transaction and closing the informational screen, the send screen (with the amount and address text boxes are still shown). Can you make it so that it goes to the Activity tab so the user can see the pending transaction? It's done in `func didFinish(_ result: ConfirmResult, in coordinator: PaymentCoordinator) {` a few lines above here.